examples/peps/utils.jl

```diff
@@ -116,15 +116,15 @@ function ITensors.apply(o⃗::Vector{ITensor}, ψ::ITensorNetwork; cutoff, maxdi
 end
 
 function flattened_inner_network(ϕ::ITensorNetwork, ψ::ITensorNetwork)
-  tn = inner(ϕ, ψ)
```

Comment:
I think `inner` should already take the proper `dag` (it should already `dag` the first tensor network `ϕ`).

Reply:
If the `dag` isn't be handled correctly by `inner` than it should be fixed there.